### ISSUE TITLE
Doc parse('now') example doesn't work.

### ISSUE THREAD

**Issue by scholer** [sdispater/pendulum#128]
With latest pip release (1.2.3):

Example, as per the doc:

```python
import pendulum

# Create testing datetime
known = pendulum.create(2001, 5, 21, 12)

# Set the mock
pendulum.set_test_now(known)

print(pendulum.now())
# '2001-05-21T12:00:00+00:00'

print(pendulum.parse('now'))  # Fails
# '2001-05-21T12:00:00+00:00'

# Clear the mock
pendulum.set_test_now()
```

Produces `ParseError`:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
~/anaconda/envs/todo/lib/python3.6/site-packages/pendulum/parsing/parser.py in _parse(self, text)
    331                 dayfirst=self._options['day_first'],
--> 332                 yearfirst=self._options['year_first']
    333             )

~/anaconda/envs/todo/lib/python3.6/site-packages/dateutil/parser.py in parse(timestr, parserinfo, **kwargs)
   1167     else:
-> 1168         return DEFAULTPARSER.parse(timestr, **kwargs)
   1169 

~/anaconda/envs/todo/lib/python3.6/site-packages/dateutil/parser.py in parse(self, timestr, default, ignoretz, tzinfos, **kwargs)
    558         if res is None:
--> 559             raise ValueError("Unknown string format")
    560 

ValueError: Unknown string format

During handling of the above exception, another exception occurred:

ParserError                               Traceback (most recent call last)
<ipython-input-44-bc93cc4d8104> in <module>()
     13 # '2001-05-21T12:00:00+00:00'
     14 
---> 15 print(pendulum.parse('now'))
     16 # '2001-05-21T12:00:00+00:00'
     17 

~/anaconda/envs/todo/lib/python3.6/site-packages/pendulum/parser.py in parse(text, **options)
     69     options['now'] = options.get('now', Global.get_test_now())
     70 
---> 71     return Parser(**options).parse(text)

~/anaconda/envs/todo/lib/python3.6/site-packages/pendulum/parser.py in parse(self, text)
     25         :rtype: mixed
     26         """
---> 27         parsed = super(Parser, self).parse(text)
     28 
     29         if not self.is_strict():

~/anaconda/envs/todo/lib/python3.6/site-packages/pendulum/parsing/parser.py in parse(self, text)
    275         :rtype: dict
    276         """
--> 277         return self.normalize(self._parse(text))
    278 
    279     def normalize(self, parsed):

~/anaconda/envs/todo/lib/python3.6/site-packages/pendulum/parsing/parser.py in _parse(self, text)
    333             )
    334         except ValueError:
--> 335             raise ParserError('Invalid date string: {}'.format(text))
    336 
    337         return {

ParserError: Invalid date string: now
```

**Comment by sdispater** [comment]
This is definitely a regression in the code.

I will take a look and get back to you.

**Comment by sdispater** [comment]
It's fixed in the new `1.2.4` release :-)
